docs/getting-started-guides/windows/index.md

```diff
@@ -99,21 +78,21 @@ To run Windows Server Containers on Kubernetes, you'll need to set up both your
 ##### Linux Host Setup
 
 1. Linux hosts should be setup according to their respective distro documentation and the requirements of the Kubernetes version you will be using. 
-2. Configure Linux Master node using steps [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md#prepare-the-master)
+2. Configure Linux Master node using steps [here](https://github.com/gkudra-msft/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/creating-a-linux-master.md)
```

Comment:
Don't put references to fork. You should change this with the official repository even if it's not merged yet in that, you know where the file would be after it will be merged.

Reply:
I thought I fixed previously... will address. thanks